### DISCUSSION TITLE
Can't create project

### DISCUSSION THREAD

**Discussion by srh98** [AntlerVC/firetable#1242] [Support / Q&A] [answered]
I'm on the pro plan. Can't create a project from the workspace. Steps followed:

- Click Workspaces
- On my workspace associated with my paid plan click "Create new project"
- gave it a name and choose Firebase
- Clicked to sign in with Google
- Clicked to continue to Google permissions
- It gets stuck after that, the modal tries to advance to the next step but then reverts to the screenshot in the attached

![Screenshot 2023-05-03 at 4 08 51 PM](https://user-images.githubusercontent.com/117376796/236070348-e9e0cad7-697d-4cc1-877d-d130c8870462.png)

**Comment by bofeiw** [comment]
Hi, looks like the auth part failed. To trouble shoot, did the popup window close itself after you clicked continue to permissions? Or it closed after choosing your Google account?

**Comment by srh98** [comment]
[Video-Wed-May-03-2023-17-38-54.webm](https://user-images.githubusercontent.com/117376796/236080062-54591d98-f5c9-4b7d-a1b3-891a4a1c6770.webm)

**Comment by srh98** [comment]
Hi! Any updates on this please? Thank you!

**Comment by bofeiw** [comment, marked as answer]
Can you try this:

1. Go to https://myaccount.google.com/permissions
2. Remove Rowy from "Third-party apps with account access"
3. Logout in the setup section, and log in again

Let me know if this helps.
